code/modules/antagonists/wizard/equipment/soulstone.dm

```diff
@@ -63,8 +63,9 @@
 	if(!ishuman(M))//If target is not a human.
 		return ..()
 	if(iscultist(M))
-		to_chat(user, "<span class='cultlarge'>\"Come now, do not capture your bretheren's soul.\"</span>")
-		return
+		if(iscultist(user))
```

Comment:
thos should be combined with line 65 above

Reply:
did this before but cyberboss didn't like it

Reply:
Oh I see what this is doing

Reply:
I don't understand why cyberboss wants it this way @Cyberboss

Reply:
I initially thought he wanted the return -1 tabbed so it would silently reject (aka cultists can't ever be sharded), otherwise it should be what JJRcop said.